webapp/client/socket.ts

```diff
@@ -0,0 +1,73 @@
+import types from '@/bindings/bindings'
+
+export type ServerCallback = (socket:Socket) => void
```

Comment:
Why is it called `ServerCalllback`, isn't it called when the websocket opens? Maybe call it `OnConnectedToServerCallback`

Reply:
Because it's used for more than one type of callbacks